class Input_Error(Exception):
    pass
